recipes/newrelic/infra-agent/amazonlinux2.yml

```diff
@@ -20,28 +20,13 @@ keywords:
 processMatch:
   - /infra/
 
-# Examine Metrics, Events, and Logging for correlated data
-# Used by the UI to determine if you've successfully configured and are ingesting data
-meltMatch:
-  events:
-    pattern:
-      - /SystemSample/
-  metrics:
-    pattern:
-      - /system.cpu.usage/
-  # logging:
-  #   pattern:
-  #     - /http/
-  #   files:
-  #     - /var\/log\/system.log
-
 validationNrql: "SELECT count(*) from SystemSample where hostname like 'HOSTNAME' SINCE 10 minutes ago"
 
+# go-task yaml definition goes here
```

Comment:
We dont have any logMatch for the regular infra-agent recipes? I thought we were going to remove the log recipe potentially, maybe another story?

Reply:
That's correct, no `logMatch` for the infra-agent recipes. Rishi had a thought to use `logMatch` for Docker use case, but my preference is to omit it from infra-agent recipes until we deem it necessary (and I think we should have a separate recipe for Docker use case). 

The CLI will kick off the Log recipe after it receives a list of `logMatch` from the recipe service, so that will still hang around.